openwisp_radius/apps.py

```diff
@@ -46,6 +46,10 @@ def ready(self, *args, **kwargs):
 
         if app_settings.SOCIAL_LOGIN_ENABLED:
             register_registration_method('social_login', _('Social login'))
+        if app_settings.SAML_LOGIN_ENABLED:
+            register_registration_method(
+                'saml', _(app_settings.SAML_REGISTRATION_METHOD_LABEL)
```

Comment:
gettext won't find this, you have to call `_()` at the line in which the string is defined